docs/API_ref.md

```diff
@@ -1,78 +1,79 @@
+| featureFlags                    | [] [string](https://pkg.go.dev/builtin#string)               | FeatureFlags defines the list of features to enable for Velero instance                                                                                                                                                                                  |
+| defaultPlugins                  | [] [string](https://pkg.go.dev/builtin#string)               | Five types of default Velero plugins can be installed:  `AWS` ,  `GCP` ,  `Azure`  and  `OpenShift` , and  `CSI` . See  [here] ( config/plugins.md) for further information.                                                                             |
+| customPlugins                   | [][CustomPlugin](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#CustomPlugin) | Used for installation of custom Velero plugins. See  [here] ( config/plugins.md) for further information.                                                                                                                                                |
+| restoreResourcesVersionPriority | [string](https://pkg.go.dev/builtin#string)                  | RestoreResourceVersionPriority represents a configmap that will be created if defined for use in conjunction with `EnableAPIGroupVersions` feature flag. Defining this field automatically add EnableAPIGroupVersions to the velero server feature flag  |
+| noDefaultBackupLocation         | [bool](https://pkg.go.dev/builtin#bool)                    | If you need to install Velero without a default backup storage location NoDefaultBackupLocation flag is required for confirmation                                                                                                                        |
+| podConfig                       | *[PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#PodConfig)              | Velero Pod specific configuration                                                                                                                                                                                                                        |
+| logLevel                       | [string](https://pkg.go.dev/builtin#string)              | Velero server’s log level (default info, use debug for the most logging). Valid options are error, warn, warning, info, debug, trace                                                                                                                                                                                                                        |
```

Comment:
"warn" and "warning" ?

Reply:
Thanks. `trace;debug;info;warning;error;fatal;panic` are valid options